src/Components/test/testassets/TestServer/Program.cs

```diff
@@ -28,7 +28,8 @@ public static void Main(string[] args)
                 ["Globalization + Localization (Server-side)"] = (BuildWebHost<InternationalizationStartup>(CreateAdditionalArgs(args)), "/subdir"),
                 ["Server-side blazor"] = (BuildWebHost<ServerStartup>(CreateAdditionalArgs(args)), "/subdir"),
                 ["Hosted client-side blazor"] = (BuildWebHost<ClientStartup>(CreateAdditionalArgs(args)), "/subdir"),
-                ["Dev server client-side blazor"] = CreateDevServerHost(CreateAdditionalArgs(args))
+                ["Dev server client-side blazor"] = CreateDevServerHost(CreateAdditionalArgs(args)),
+                ["Protected browser storage"] = (BuildWebHost<ProtectedBrowserStorageStartup>(CreateAdditionalArgs(args)), "/subdir")
```

Comment:
Having too many different startup classes will make it harder to work with the tests if they keep accumulating, as you'll have to figure out which startup class is meant to be used with which tests when running them manually.

If we do need a new startup class here (instead of, say, merging this into the existing "Server-side blazor" one), could you add something into the test components such as `ProtectedBrowserStorageInjectionComponent.razor`, e.g.,

```html
<h2>Note: This component can only be run when launched with the <strong>Protected browser storage</strong> startup class.</h2>
```

Reply:
Agree with @SteveSandersonMS, I would just add the protected browser storage to some existing startup class and use that as the base.